flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/utils/ExecNodeMetadataUtilTest.java

```diff
@@ -319,23 +311,19 @@ public void testStreamExecNodeJsonSerdeCoverage() {
         @ExecNodeMetadata(
                 name = "dummy-node",
                 version = 1,
-                consumedOptions = {"option1", "option2"},
+                consumedOptions = {"option11", "option22", "option33"},
                 minPlanVersion = FlinkVersion.v1_13,
                 minStateVersion = FlinkVersion.v1_13),
         @ExecNodeMetadata(
                 name = "dummy-node",
                 version = 2,
-                consumedOptions = {"option11", "option22"},
+                consumedOptions = {"option111", "option222", "option333"},
                 minPlanVersion = FlinkVersion.v1_14,
                 minStateVersion = FlinkVersion.v1_14),
         @ExecNodeMetadata(
                 name = "dummy-node",
                 version = 3,
-                consumedOptions = {
-                    "table.exec.state.ttl",
-                    "table.exec.sink.not-null-enforcer",
-                    "table.exec.deduplicate.mini-batch.compact-changes.enabled" // deprecated key
-                },
+                consumedOptions = {"option1", "option2", "option3-deprecated", "option4-fallback"},
```

Comment:
let's represent the reality here, the version 1 uses a deprecated key that is a new key in version 2 or 3?